.github/workflows/setupmc.yml

```diff
@@ -0,0 +1,27 @@
+            - name: Setup Minecraft
+  # You may pin to the exact commit or the version.
+  # uses: mcenv/setup-minecraft@2d092242881552b8ee463d9b4687bd38ef58a396
+  uses: mcenv/setup-minecraft@v3.3.2
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (bug_risk):** Inconsistent versions of setup-minecraft action used.

The workflow uses 'mcenv/setup-minecraft@v3.3.2' and later 'mcenv/setup-minecraft@v3'. It's important to use consistent versions across the workflow to avoid unexpected behavior.